watertap/property_models/seawater_prop_pack.py

```diff
@@ -63,17 +63,38 @@
 )
 import idaes.core.util.scaling as iscale
 from watertap.core.util.scaling import transform_property_constraints
+from pyomo.common.config import ConfigValue, In
+from enum import Enum, auto
 
 # Set up logger
 _log = idaeslog.getLogger(__name__)
 
 
+class DiffusCalculationType(Enum):
+    isothermal = auto()
+    nonisothermal = auto()
+
+
 @declare_process_block_class("SeawaterParameterBlock")
 class SeawaterParameterData(PhysicalParameterBlock):
     """Parameter block for a seawater property package."""
 
     CONFIG = PhysicalParameterBlock.CONFIG()
 
+    CONFIG.declare(
+        "diffus_calc_type",
```

Comment:
```suggestion
        "diffus_calculation",
```
Just for naming consistency, since MCAS has a config option related to calculating diffusivity, might as well use the same name.

Reply:
Thank you! The naming was what I was unsure about.